packages/html/src/index.ts

```diff
@@ -1,31 +1,37 @@
-export { Interactor, InteractorSpecificationBuilder, InteractorConstructor, InteractorBuilder, InteractorSpecification } from './specification';
-export { Interaction, ReadonlyInteraction, isInteraction } from './interaction';
-export { createInteractor } from './create-interactor';
-export { createInspector } from './inspector'
-export { Page } from './page';
-export { App } from './app';
-export { read } from './read';
-export { perform } from './perform';
-export { fillIn } from './fill-in';
-export { focused, focus, blur } from './focused';
-export { isVisible } from 'element-is-visible';
-export { Matcher } from './matcher';
+export {
+  Interactor,
+  InteractorSpecificationBuilder,
+  InteractorConstructor,
+  InteractorBuilder,
+  InteractorSpecification,
+} from "./specification";
+export { Interaction, ReadonlyInteraction, isInteraction } from "./interaction";
+export { createInteractor } from "./create-interactor";
+export { createInspector } from "./inspector";
+export { Page } from "./page";
+export { App } from "./app";
+export { read } from "./read";
+export { perform } from "./perform";
+export { fillIn } from "./fill-in";
+export { focused, focus, blur } from "./focused";
+export { isVisible } from "element-is-visible";
```

Comment:
Most of the changes in this file seem completely irrelevant to improving click emulation. If we want to add linting or formatting tools, this should be done as a separate PR and in an automated fashion so that everybody will stay in sync.

Please review your commits before submitting to make sure that they directly pertain to the change you are proposing.